pkg/controller/populators/util.go

```diff
@@ -17,12 +32,16 @@ const (
 
 	// errCreatingPVCPrime provides a const to indicate we failed to create PVC prime for population
 	errCreatingPVCPrime = "ErrCreatingPVCPrime"
-	// createdPVCPrimeSucceesfully provides a const to indicate we created PVC prime for population
-	createdPVCPrimeSucceesfully = "CreatedPVCPrimeSucceesfully"
 
-	// AnnSelectedNode annotation is added to a PVC that has been triggered by scheduler to
-	// be dynamically provisioned. Its value is the name of the selected node.
-	AnnSelectedNode = "volume.kubernetes.io/selected-node"
```

Comment:
still need in this commit this const and also the function GetSelectedNodeName

Reply:
Should be already :+1: